### DISCUSSION TITLE
ERRO 403 (Forbidden) ao executar o deploy de MY-CUSTOM-PRIVATE-PACKAGE

### DISCUSSION THREAD

**Discussion by luizgoebel** [balta-io/forum#226] [Tira Dúvidas] [answered]
![image](https://user-images.githubusercontent.com/72111314/212747047-7bd697f3-e511-4db2-9368-fa95fa0310bc.png)
![image](https://user-images.githubusercontent.com/72111314/212747079-eb6646f6-bf2b-4520-bf1f-810b16c6fec6.png)
![image](https://user-images.githubusercontent.com/72111314/212747103-683d2fcb-6447-4bd0-a418-5ce13fce7a8e.png)

**Comment by andrebaltieri** [comment, marked as answer]
Duplicado
https://github.com/balta-io/forum/discussions/123

**Reply by luizgoebel** [reply]
Mesmo mudando o nome do pacote, continua gerando erro...

![image](https://user-images.githubusercontent.com/72111314/212904244-3f87b3b5-c3bd-4898-a00d-2fd7c7f4105d.png)
![image](https://user-images.githubusercontent.com/72111314/212904360-a8a9a60d-ac92-4672-806a-4d6690d6986f.png)

**Reply by luizgoebel** [reply]
procurei várias opções de contornar o erro, todas sem sucesso, continua o 403

**Reply by andrebaltieri** [reply]
@luizgoebel, conforme comentei na sua outra dúvida, você criou o NUGET_TOKEN no repositório do GitHub(https://github.com/<<SEU USUARIO>>/<<SEU REPOSITORIO>>/settings/secrets/actions) com o valor que obteve do Nuget.org?

O erro que está sendo apresentado é referente a autenticação... provavelmente por que ele não conseguiu obter o token, ou o token gerado tem nome de pacote diferente do seu package Id.

**Comment by crevelloni** [comment]
"...ou o token gerado tem nome de pacote diferente do seu package Id."
Essa resposta salvou minha noite, agora vou dormir tranquilo kkk
